pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -58,14 +58,16 @@ import (
 )
 
 const (
-	mysqlPort    = "3306"
-	postgresPort = "5432"
-	mongoPort    = "27017"
-	redisPort    = "6379"
-	amqpPort     = "5672"
-	kafkaPort    = "9092"
-	http2Port    = "9090"
-	grpcPort     = "9091"
+	mysqlPort      = "3306"
```

Comment:
nit: maybe add a comment here (can be a copy paste from your commit note) that currently all those protocol classifications are not supported on other platforms than linux, hence they are located in this file